src/main/kotlin/no/nav/dagpenger/regel/api/db/SubsumsjonStore.kt

```diff
@@ -1,15 +1,20 @@
 package no.nav.dagpenger.regel.api.db
 
+import no.finn.unleash.Unleash
 import no.nav.dagpenger.regel.api.models.*
 import java.time.ZoneId
 import java.time.ZonedDateTime
 
 interface SubsumsjonStore {
```

Comment:
kunne vi latt SubsumsjonsStore extenda Unleash her for å slippe å la unleash bli med i alle kallene nedover

ala 

```interface SubsumsjonStore : Unleash {
...
val internBehov = InternBehov.fromBehov(behov, behandlingsId, isEnabled("dp.korona", false))
... 

```

og i PostgresSubsumsjonStore

```
internal class PostgresSubsumsjonStore(private val dataSource: DataSource, private val unleash: Unleash) : SubsumsjonStore, HealthCheck, Unleash by unleash {
```

Da slipper en å passe unleash gjennom hele stacken

Reply:
Enig at det er mye unleash vi får gjennom hele stacken nå

Egentlig føler jeg at oppretting av internbehov-logikken burde trekkes ut av Subsumsjonsstore, ettersom den har like relevant for databasen som for kafka?

Generelt er ting litt sausa sammen i regel-api nå

Reply:
Kunne kanskje ekstrahert toPacket fra Internbehov og skille mellom http lag og kafka/db lag

Reply:
Vi merger nå for å få testa minsteinntekt og sats, men vi tar en refaktoreringsrunde i mårra